LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 E. Dunham
```

Comment:
Not sure you're the Copyright holder, maybe change it to "The Cobalt contributors"? :wink: